package.json

```diff
@@ -90,5 +90,8 @@
     "q": "1.4.1",
     "u2f-api-polyfill": "0.4.1",
     "underscore": "1.8.3"
+  },
+  "optionalDependencies": {
+    "fsevents": "*"
```

Comment:
It is interesting that this is required - usually the dependencies we pull in should have it listed as optional. I'm okay with it if for some reason it's necessary.